client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/StatisticsConfigurationBuilder.java

```diff
@@ -31,37 +31,66 @@
       super(builder);
    }
 
+   /**
+    * Enables or disables client-side statistics collection
+    *
+    * @param enabled whether to enable client-side statistics
+    */
    public StatisticsConfigurationBuilder enabled(boolean enabled) {
       attributes.attribute(ENABLED).set(enabled);
       return this;
    }
 
+   /**
+    * Enables client-side statistics collection
+    */
    public StatisticsConfigurationBuilder enable() {
       return enabled(true);
    }
 
+   /**
+    * Disables client-side statistics collection
+    */
    public StatisticsConfigurationBuilder disable() {
       return enabled(false);
    }
 
+   /**
+    * Enables or disables exposure of client-side statistics over JMX
+    */
    public StatisticsConfigurationBuilder jmxEnabled(boolean enabled) {
       attributes.attribute(JMX_ENABLED).set(enabled);
       return this;
    }
 
+   /**
+    * Enables exposure of client-side statistics over JMX
+    */
    public StatisticsConfigurationBuilder jmxEnable() {
       return jmxEnabled(true);
    }
 
+   /**
+    * Disables exposure of client-side statistics over JMX
+    */
    public StatisticsConfigurationBuilder jmxDisable() {
       return jmxEnabled(false);
    }
 
+   /**
+    * Sets the JMX domain name with which MBeans are exposed. Defaults to org.infinispan
```

Comment:
What about adding a link to {@link StatisticsConfiguration#JMX_DOMAIN } ?  to point where the default value is hard-coded